actors.md

```diff
@@ -471,11 +450,10 @@ func CommitSector(sectorID SectorID, commD, commR, commRStar []byte, proof SealP
 	// make sure the miner has enough collateral to add more storage
 	coll = CollateralForSector(miner.SectorSize)
 
-	if coll < miner.Collateral {
+  if coll < vm.MyBalance() - miner.ActiveCollateral {
```

Comment:
Note: I see that the [`SlashStorageFault`](https://github.com/filecoin-project/specs/pull/312/files#diff-8cd5872b87e5b64500c99ea50b3a9241R634) method consumes the `ConsensusCollateral` field on the miner. I don't see that field specified anywhere. Should this check be rewritten as:

```
if coll < vm.MyBalance() - miner.ActiveCollateral - miner.ConsensusCollateral
```

?